### DISCUSSION TITLE
关于新版本界面

### DISCUSSION THREAD

**Discussion by murphy194** [MaaAssistantArknights/MaaAssistantArknights#4821] [Ideas]
4.17.1版本的新界面相比于老版本的界面更加的简洁了，但这样也让部分功能的操作设置增加了一步甚至更多，我认为可以加上一个新版本和老版本的界面互换功能，让使用者可以体验到两个版本的不同，感受两个版本界面的优势所在，让用户能够选择自己较为喜欢的版本界面

**Comment by gsxgmpkt9020** [comment]
离。。。。。。离大谱(>_<)

**Comment by ABA2396** [comment]
新版本在刷理智的高级设置中加了 `显示基建计划` 应该和原来差不多了吧
